content/docs/2.15/scalers/beanstalkd.md

```diff
@@ -0,0 +1,84 @@
++++
+title = "Beanstalkd"
+availability = "v2.15+"
```

Comment:
Can you please update this to 2.16 and move to the new folder please?

Reply:
Thanks for the review. All updated to 2.16.